externs/olx.js

```diff
@@ -4361,7 +4378,6 @@ olx.source.TileImageOptions.prototype.wrapX;
  *            cacheSize: (number|undefined),
  *            format: (ol.format.Feature|undefined),
  *            logo: (string|olx.LogoOptions|undefined),
- *            opaque: (boolean|undefined),
```

Comment:
Good question. The option will not hurt if someone has it in their code. And it did not work properly previously. Happy to add it if you think. Do you?